data-sources/ray/config.yml

```diff
@@ -0,0 +1,17 @@
+id: ray
+displayName: Ray
+description: Identify and optimize Ray performance with New Relic Ray monitoring.
```

Comment:
```suggestion
description: Monitor the performance of Ray using New Relic monitoring.
```

Reply:
Hi @seemantk ,
As per your suggestion, I have made the changes. Could you please review it?